### ISSUE TITLE
Default node attributes

### ISSUE THREAD

**Issue by bard** [cpettitt/graphlib-dot#30]
I see that it's possible to set default graph attributes in the generated graphviz code by calling `setGraph()`:

```
const graph = new Graph()
graph.setGraph({rankdir: 'LR'})
dot.write(graph)
```

Results in:

```
strict digraph {
  rankdir=LR;
}
```

I'd like to do the same for default node attributes in the generated graphviz code. I've found that creating a node called "node" will do the trick:

```
const graph = new Graph()
graph.setNode('node', { shape: 'box' })
```

Results in:

```
strict digraph {
  node [shape=box]
}
```

But, of course, it also creates an extra node named "node" in the graph as seen by graphlib.

Can the same be achieved without creating that node?

**Comment by cpettitt** [comment]
If you're using the graph API directly, then you can use this: https://github.com/cpettitt/graphlib/wiki/API-Reference#setDefaultNodeLabel. If you're using the DOT parser, it should handle node defaults.

**Comment by bard** [comment]
I've edited the question above for clarity. I'm using the graph API directly. 

Unfortunately, calling `setDefaultNodeLabel` in my case doesn't work because nodes have attributes already, so the default gets ignored.

Fortunately, I came across https://github.com/cpettitt/dagre-d3 by you-know-whom that lets me take graphviz out of the loop entirely in favor of d3 which was the eventual goal anyway.

Thanks!
